### ISSUE TITLE
testify version mismatches in go.mod and vendor/modules.txt

### ISSUE THREAD

**Issue by linqing2004** [terraform-providers/terraform-provider-oci#1176]
When we build with mod=vendor, we got the following error.
I fixed it by changing the go.mod version to 1.6.1. Not sure if it is the correct fix.
Please get this fixed in the next release.

Thanks,
---------------------------------------------------------------------------
export GO111MODULE=on
export GOFLAGS=-mod=vendor
go install
BUILDSTDERR: go: inconsistent vendoring in /builddir/build/BUILD/terraform-provider-oci-3.93.0:
BUILDSTDERR: 	github.com/stretchr/testify@v1.3.0: is explicitly required in go.mod, but vendor/modules.txt indicates github.com/stretchr/testify@v1.6.1
------------------------------------------------------------------------------------

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Configuration Files

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. 
# Please remove any sensitive information from configuration files before sharing them. 
```

### Steps to Reproduce

1. `terraform apply`

**Comment by jotruon** [comment]
Hi @linqing2004,
Thanks for reporting this inconsistent vendoring.
The step you are referring is correct and it is in the make file: https://github.com/terraform-providers/terraform-provider-oci/blob/master/GNUmakefile#L16-L24

**Comment by abhilash-av** [comment]
This is now fixed in Terraform Provider Oci V3.94.0
